scripts/spark2diffs/RapidsMeta.diff

```diff
@@ -0,0 +1,394 @@
+2c2
+<  * Copyright (c) 2022, NVIDIA CORPORATION.
+---
+>  * Copyright (c) 2019-2021, NVIDIA CORPORATION.
+23c23
+< import org.apache.spark.sql.catalyst.expressions.{Attribute, AttributeReference, BinaryExpression, ComplexTypeMergingExpression, Expression, String2TrimExpression, TernaryExpression, UnaryExpression, WindowExpression, WindowFunction}
+---
+> import org.apache.spark.sql.catalyst.expressions.{Attribute, AttributeReference, BinaryExpression, ComplexTypeMergingExpression, Expression, QuaternaryExpression, String2TrimExpression, TernaryExpression, UnaryExpression, WindowExpression, WindowFunction}
+25a26,27
+> import org.apache.spark.sql.catalyst.trees.TreeNodeTag
+> import org.apache.spark.sql.connector.read.Scan
+27c29
+< import org.apache.spark.sql.execution.aggregate._
+---
+> import org.apache.spark.sql.execution.aggregate.BaseAggregateExec
+30c32
+< import org.apache.spark.sql.execution.window.WindowExec
+---
+> import org.apache.spark.sql.rapids.{CpuToGpuAggregateBufferConverter, GpuToCpuAggregateBufferConverter}
+54a57
+>   val gpuSupportedTag = TreeNodeTag[Set[String]]("rapids.gpu.supported")
+67a71,72
+>  * @tparam OUTPUT when converting to a GPU enabled version of the plan, the generic base
+>  *                    type for all GPU enabled versions.
+69c74
+< abstract class RapidsMeta[INPUT <: BASE, BASE](
+---
+> abstract class RapidsMeta[INPUT <: BASE, BASE, OUTPUT <: BASE](
+72c77
+<     val parent: Option[RapidsMeta[_, _]],
+---
+>     val parent: Option[RapidsMeta[_, _, _]],
+88c93
+<   // val childScans: Seq[ScanMeta[_]]
```

Comment:
There's a mix of deleting stuff 2.x explain does not need vs. commenting it out.  Was there a reason this file has a lot of comments instead of deletions as was done most places elsewhere?

Reply:
I'll remove these